### DISCUSSION TITLE
好冷清啊。

### DISCUSSION THREAD

**Discussion by ForteScarlet** [ForteScarlet/simpler-robot#41] [不要在这里发帖子了]
真的有人在看吗？莫非一直以来我只是在自娱自乐..😟

**Comment by ljnForLove1314** [comment]
你不是一个人在战斗💪

**Comment by 194395406** [comment]
有没有获取群员信息的api啊？我看了一下文档好像没有这种api，只有mirai插件的获取群文件的api

**Reply by ForteScarlet** [reply]
> 有没有获取群员信息的api啊？我看了一下文档好像没有这种api，只有mirai插件的获取群文件的api

我猜你只看了mirai组件下的 额外API，你可以参考 [送信器](https://www.yuque.com/simpler-robot/simpler-robot-doc/sp4s9g)

**Reply by ForteScarlet** [reply]
而且假如下次还有疑问的话，建议另开一个讨论主题或issue 🐱

**Reply by 194395406** [reply]
吼 谢谢 是我没有看仔细 这就去研究研究

**Comment by 1B9ear** [comment]
怎么加QQ群呀
